helm/hpcc/values.schema.json

```diff
@@ -3045,6 +3045,31 @@
       },
       "additionalProperties": false
     },
+    "sasha-debugplane-housekeeping": {
+      "type": "object",
+      "allOf": [{ "$ref": "#/definitions/sashacommon" }],
+      "properties": {
+        "expiryDays": {
+          "type": "integer",
+          "description": "Default number of days passed that post-mortem files are deleted after their associated workunit was completed",
+          "default": "14"
+        },
+        "disabled": {},
+        "interval": {},
+        "service": {},
```

Comment:
I can't remember how these work exactly, I think they're necessary if you want to inherit them in the schema,
but "service" we don't want as part of this sasha component.
Others that are not relevant:
- throttle
- plane
- egress